apis/telemetry/v1alpha1/logpipeline_validation_test.go

```diff
@@ -0,0 +1,379 @@
+package v1alpha1
+
+import (
+	"github.com/stretchr/testify/require"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"testing"
+)
+
+func TestContainsNoOutputPlugins(t *testing.T) {
+	logPipeline := &LogPipeline{
+		Spec: LogPipelineSpec{
+			Output: Output{},
+		}}
+
+	vc := getLogPipelineValidationConfig()
+	result := logPipeline.validateOutput(vc.DeniedOutPutPlugins)
+
+	require.Error(t, result)
+	require.Contains(t, result.Error(), "no output is defined, you must define one output")
```

Comment:
same question as in [logpipeline_validation.go](https://github.com/kyma-project/telemetry-manager/pull/76/files#diff-bb7dcb9543c2aa6f9afd6c7cdb7a5fab37fa28382dda1560b0a562c2a3ba0d93) line 51/54.